misc/services/mobile-webui/mobile-webui-frontend/src/reducers/wfProcesses_status/picking.js

```diff
@@ -148,6 +234,30 @@ const normalizePickingLines = (lines) => {
       ...line,
       steps: line.steps.reduce((accum, step) => {
         accum[step.pickingStepId] = step;
+        accum[step.pickingStepId].altSteps = {};
+        accum[step.pickingStepId].altSteps.qtyToPick = 0;
+        accum[step.pickingStepId].altSteps.genSteps = {};
```

Comment:
u could merge these 3 lines in 1 object assignment.

something like: 
accum[step.pickingStepId].altSteps = { qtyToPick: 0, genSteps: {} };

Reply:
also, i think it's IMPORTANT to consider the current qtys that u get from backend and create the proper "genSteps" from it.
Else, if the store gets clear then the application will show the steps as not picked.